Application/EdFi.Ods.WebApi/appsettings.json

```diff
@@ -12,6 +12,7 @@
     },
     "Tenants": {
     },
+    "StartUpClassName": "",
```

Comment:
Change the capitalization of the name in this setting to `StartupClassName` (Startup is treated as one word, not two).